### ISSUE TITLE
Add support for Logtail

### ISSUE THREAD

**Issue by ovidiuenache** [Seldaek/monolog#1799]
I am wondering whether adding a handler for [Logtail](https://betterstack.com/logtail) would be of interest or not. I am using it in one of my projects and it doesn't have a built-in handler. I am happy to come up with a PR if you'd like such an addition.

**Comment by mabar** [comment]
Hello,
Logtail wrote their own, curl-based integration https://github.com/logtail/monolog-logtail
There is also my simplified PSR-based integration https://github.com/orisai/monolog-logtail

**Comment by ovidiuenache** [comment]
I see.

Is there a moment when such a handler becomes part of the main repository?

I'm asking this because I see several handlers for numerous log management systems (e.g. Logentries, Loggly, etc.) being added to the main repository. At some point, they were also accessible only through separate packages.

**Comment by Seldaek** [comment]
Nope, we stopped adding new handlers unless they are completely general purpose with no external deps. It's too complex for me to maintain all these services without any knowledge or ways to test.

So IMO this is better as it is.
